src/main/java/org/broadinstitute/dsm/route/PatchRoute.java

```diff
@@ -549,17 +551,18 @@ private NameValue setAdditionalValue(String additionalValue, @NonNull Patch patc
     }
 
     private void triggerParticipantEvent(DDPInstance ddpInstance, Patch patch, Value action){
-        inTransaction((conn) -> {
-            Optional<EventTypeDto> eventType = EventTypeDao.getEventTypeByEventTypeAndInstanceId(conn, action.getName(), ddpInstance.getDdpInstanceId());
+            Optional<EventTypeDto> eventType = eventTypeDao.getEventTypeByEventTypeAndInstanceId(action.getName(), ddpInstance.getDdpInstanceId());
             if (eventType != null && !eventType.isEmpty()) {
```

Comment:
I think it does not make sense to check whether `eventType` is null or not, since method `getEventTypeByEventTypeAndInstanceId` will return either empty `Optional` or filled with some data, thus you could use directly `if (eventType.isPresent())` or `eventType.ifPresent(eventTypeDto -> {//logic here})`.